source/module_elecstate/potentials/efield.cpp

```diff
@@ -38,6 +38,9 @@ ModuleBase::matrix Efield::add_efield(const UnitCell& cell,
     ModuleBase::TITLE("Efield", "add_efield");
     ModuleBase::timer::tick("Efield", "add_efield");
 
+    // set the parameters
+    autoset(cell);
```

Comment:
just a comment but not needed to change: does this place where autoset function is inserted, supports dynamically adjust respect to geometry change? Say during relax of slab systems, the induced dipole field would change. Is this case taken into consideration now?

Reply:
Please refer to the linked issue. It seems that VASP also ONLY set it once.

Reply:
These parameters will not change during the relaxation of slab systems.